mautic/mautic
# Dynamics: Fix process error

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
AbstractIntegration retunr from some time error as array https://github.com/mautic/mautic/blob/973e9ca377032576472d2818fd3e9882f2d76f4c/app/bundles/PluginBundle/Integration/AbstractIntegration.php#L829

This PR fix process error in Dynamics



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  You catch error during sync. Then process error not finish with fatal error
